components/style/servo/selector_parser.rs

```diff
@@ -436,6 +439,7 @@ impl<'a, 'i> ::selectors::Parser<'i> for SelectorParser<'a> {
             "active" => Active,
             "any-link" => AnyLink,
             "checked" => Checked,
+        "defined" => Defined,
```

Comment:
indentation here seems broken.

Reply:
That's weird! I ran fmt and tidy and didn't look closely at the result; maybe it's a bug in fmt.